files/dhcp/sethostname

```diff
@@ -1,20 +1,13 @@
-#!/bin/bash
-# This script is to update hostname of the system.
+case $reason in
+    BOUND|RENEW|REBIND|REBOOT)
+        current_host_name=`hostname -s`
+        if [ "$current_host_name" != "$new_host_name" ] && [ -n "$new_host_name" ]
+        then
+            echo $new_host_name > /etc/hostname
+            hostname -F /etc/hostname
+            sed -i "/\s$current_host_name$/d" /etc/hosts
+        echo "127.0.0.1 $new_host_name" >> /etc/hosts
```

Comment:
please make make sure the indent is matching other lines.

---
In reply to: [100711735](https://github.com/Azure/sonic-buildimage/pull/288#discussion_r100711735) [](ancestors = 100711735)